hw/dv/tools/dvsim/sim.mk

```diff
@@ -120,10 +120,28 @@ cov_unr_build: gen_sv_flist
 	@echo "[make]: cov_unr_build"
 	cd ${sv_flist_gen_dir} && ${cov_unr_build_cmd} ${cov_unr_build_opts}
 
-cov_unr: cov_unr_build
+cov_unr_vcs: cov_unr_build
 	@echo "[make]: cov_unr"
 	cd ${sv_flist_gen_dir} && ${cov_unr_run_cmd} ${cov_unr_run_opts}
 
+cov_unr_xcelium: 
+ifeq ("$(wildcard $(cov_merge_db_dir))", "")
+	@echo "NO COVERAGE DATABASE DEFINED"
+else
+	@echo "[make]: cov_unr"
+	mkdir -p ${cov_unr_dir}
+	cd ${cov_unr_dir}; ${cov_unr_run_cmd} ${cov_unr_run_opts}
+endif
+
+cov_unr_merge:
+	 ${cov_merge_cmd} -init ${cov_unr_dir}/jgproject/sessionLogs/session_0/unr_imc_coverage_merge.cmd 
```

Comment:
This is fine but (possibly as a follow-up) maybe it would be better to generate the command line in the hjson code (so the "jgproject" stuff doesn't appear in this common Makefile). If we do that, we can actually do even better and define `cov_merge_cmd` for VCS as well (as e.g. just a call to `true`). That way, we won't need the if/else when defining `cov_unr`.

Reply:
let have this as a planned update. :D